internal/utils/misc.go

```diff
@@ -170,6 +170,7 @@ var (
 	CurrBranchPath        = filepath.Join(SupabaseDirPath, ".branches", "_current_branch")
 	MigrationsDir         = filepath.Join(SupabaseDirPath, "migrations")
 	FunctionsDir          = filepath.Join(SupabaseDirPath, "functions")
+	EmailTemplatesDir     = filepath.Join(SupabaseDirPath, "templates", "email")
```

Comment:
```suggestion
	EmailTemplatesDir     = filepath.Join(SupabaseDirPath, "auth", "email")
```

Seems more appropriate to nest under `auth` since the config object is also nested under auth.